pkg/controller/elasticsearch/bootstrap/bootstrap.go

```diff
@@ -6,8 +6,11 @@ package bootstrap
 
 import (
 	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
-	"github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/observer"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/client"
 	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
+
+	"context"
+
```

Comment:
nit: spaces in imports

Reply:
This is what IntelliJ produces when clicking on "Optimize imports". I tried sticking all imported lines together then reapplying the optimization. This is what I get:

```go
import (
	"context"

	logf "sigs.k8s.io/controller-runtime/pkg/log"

	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
	"github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/client"
	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
)

```